packages/remix-dev/vite/build.ts

```diff
@@ -139,18 +139,21 @@ async function getServerBuilds({
       }
       serverBundlesManifest.routeIdToServerBundleId[route.id] = bundleId;
 
-      let serverBundleDirectory = path.join(serverBuildDirectory, bundleId);
+      let relativeServerBundleDirectory = path.relative(
+        rootDirectory,
+        path.join(serverBuildDirectory, bundleId)
+      );
       let serverBuildConfig = serverBuildConfigByBundleId.get(bundleId);
       if (!serverBuildConfig) {
         serverBundlesManifest.serverBundles[bundleId] = {
           id: bundleId,
           file: normalizePath(
-            path.join(serverBundleDirectory, serverBuildFile)
+            path.join(relativeServerBundleDirectory, serverBuildFile)
```

Comment:
This file is a case where we _want_ it to be relative since it's a root-relative manifest file.